e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/generate-keying-sheet.spec.js

```diff
@@ -168,7 +168,7 @@ context('PDC_RECONCILE users can generate keying data', () => {
      * difference between the current and previous
      * utilisation
      */
-    const expectedPrincipalBalanceAdjustment = '20,000.00'; // 200000 - 180000, DECREASE
+    const expectedPrincipalBalanceAdjustment = '128,000.00'; // 200000 - (180000 * 0.4), DECREASE
```

Comment:
Just leaving a comment based on what we spoke about on the call:

The previous test read:
We have a saved utilisation of 200,000 (where the bank had previously uploaded 200,000)
Bank uploads a new utilisation of 180,000
We calculate a difference of 20,000

This one now reads
We have a saved utilisation of 200,000 (where thebank will have uploaded in previous month 200,000/0.4 = 500000
Bank uploads a utilisation of 180,000 which is 72,000 UKEF utilisation
the decrease is 128,000

I think after thinking this through it is absolutely fine it's all fake numbers, I was getting hung up on what a 'real life case' might look